tests/c_cpp/src/main.c

```diff
@@ -0,0 +1,7 @@
+int main()
+{
+    int array[5];
+    int i = -1;
+    int j = array[i];
```

Comment:
Codacy found an issue: [Variable 'j' is assigned a value that is never used.](https://app.codacy.com/gh/cnescatlab/sonar-scanner/pullRequest?prid=6081579)